### PR TITLE
Expire flyte_idt cookie at logout

### PR DESCRIPTION
**Pull request by iaroslav-ciupin**
# TL;DR
As discussed with @EngHabu, we need to expire flyte_idt cookie containing ID token as well during the logout.
Also, I've run `make update_boilerplate`.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
_Remove the '*fixes*' keyword if there will be multiple PRs to fix the linked issue_

fixes https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: boilerplate/flyte/end2end/Makefile [hunk 1]
```diff
@@ -4,6 +4,8 @@
 # TO OPT OUT OF UPDATES, SEE https://github.com/flyteorg/boilerplate/blob/master/Readme.rst
 
 .PHONY: end2end_execute
+end2end_execute: export FLYTESNACKS_PRIORITIES ?= P0
```
Comment (katrogan) [inline boilerplate/flyte/end2end/Makefile:7]: why this change?

Comment (katrogan) [reply boilerplate/flyte/end2end/Makefile:7]: ah can you update the PR description to include you've synced boilerplate?

**Review by katrogan — APPROVED** [review]

**Review by EngHabu — APPROVED** [review]
Thank you!

**Review by katrogan — APPROVED** [review]
